articles/logs/streams/datadog.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Datadog Event Log Streams
+description: Learn about Datadog Event Log Streams and how they let you export your events in near real-time to Datadog.
+toc: false
+topics:
+ - logs
+ - streams
+ - event-streams
+contentType: how-to
+---
+
+# Datadog Event Log Streams
+
+Datadog is the essential monitoring platform for cloud applications. It brings together data from servers, containers, databases, and third-party services to make your stack entirely observable.
```

Comment:
What Datadog subscription do I need to leverage this? Is it Datadog Log Management? Or Datadog Infrastructure? We should clarify.

https://www.datadoghq.com/pricing/

Reply:
Excellent point 👍 I've added a note about the plan they need in order for this integration to work